src/main/scala/dx/translator/cwl/CallableTranslator.scala

```diff
@@ -0,0 +1,72 @@
+package dx.translator.cwl
+
+import java.nio.file.Path
+
+import dx.api.DxApi
+import dx.core.ir.{Application, Callable, CallableAttribute, ExecutableKindApplet, Parameter, ParameterAttribute, RuntimeRequirement, Value}
+import dx.core.languages.cwl.CwlUtils
+import dx.translator.{ReorgSettings, RunSpec}
+import dx.cwl.{BooleanValue, CommandInputParameter, CommandLineTool, CommandOutputParameter, DockerRequirement, LongValue, Process, SchemaDefRequirement, ToolTimeLimitRequirement, WorkReuseRequirement, Workflow}
+import dx.translator.RunSpec.{ContainerImage, IgnoreReuseRequirement, TimeoutRequirement}
+
+
+case class CallableTranslator(cwlBundle: CwlBundle,
+                              locked: Boolean,
+                              defaultRuntimeAttrs: Map[String, Value],
+                              reorgAttrs: ReorgSettings,
+                              dxApi: DxApi = DxApi.get) {
+
+
+  def translateCallable(callable: Process, sourceFile: Path): Callable = {
+    callable match {
+      case tool: CommandLineTool =>
+        val taskTranslator = CwlToolTranslator.apply(tool, sourceFile)
```

Comment:
No need for the `.apply`. Calling `CwlToolTranslator(tool, sourceFile)` causes the `apply` function to be called.